oxidation/libparsec_python/src/protocol/cmd.rs

```diff
@@ -0,0 +1,59 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
+
+use pyo3::import_exception;
+use pyo3::prelude::*;
+use pyo3::types::{PyBytes, PyType};
+
+use parsec_api_protocol::{authenticated_cmds, invited_cmds};
+
+import_exception!(parsec.api.protocol, ProtocolError);
+
+#[pyclass]
+#[derive(PartialEq, Clone)]
+pub(crate) struct AuthenticatedCmdReq(pub authenticated_cmds::AnyCmdReq);
```

Comment:
Don't want to call that `°AuthenticatedAnyCmdReq` ?

Reply:
Why not